Makefile

```diff
@@ -54,7 +54,7 @@ BASE_CFLAGS := -O2 -Wall -g -Wundef -Werror=strict-prototypes -fno-common -fno-P
 	-Wsign-compare -Wunused-parameter -Wno-multichar \
 	-ffreestanding -fpic -ffunction-sections -fdata-sections \
 	-nostdinc -isystem $(shell $(CC) -print-file-name=include) -isystem sysinc \
-	-fno-stack-protector -mstrict-align -march=armv8.2-a \
+	-fno-stack-protector -mstrict-align -Wa,-march=armv8.2-a \
```

Comment:
I don't think this is enough for armv8-a under all circumstances. I think you have to explicitly set `-march=armv8-a` and add `-mno-outline-atomics` to make sure we end up with base armv8-a